hoot-core-test/src/test/cpp/hoot/core/conflate/splitter/DualWaySplitterTest.cpp

```diff
@@ -75,6 +99,10 @@ class DualWaySplitterTest : public CppUnit::TestFixture
 
     OsmXmlWriter writer;
     writer.write(after, "test-output/DualWaySplitterTest.osm");
+
+    HOOT_FILE_EQUALS(
+      "test-output/DualWaySplitterTest.osm",
+      "test-files/DualWaySplitterTestExpected.osm");
```

Comment:
This is the one that gets me, `allTest` would run, an output file was written but we never did ANYTHING to test if the splitter worked, or did anything.  The file could have been blank for all we cared as long as the software didn't crash the test passed.

Reply:
Not crashing is the first step...